### PR TITLE
fix(icons): esim and funnel icons

### PR DESCRIPTION
**Pull request by yceballost**
The 98% of changes are caused by Figma instability with svgs. Nothing important.

The real changes is: Added funnel and esim icons to Telefónica set (in 3 weights)

### CODE REVIEW

File: src/generated/mistica-icons/icon-esim-light.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-esim-light.tsx:1]: the real change

File: src/generated/mistica-icons/icon-esim-regular.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-esim-regular.tsx:1]: the real change

File: src/generated/mistica-icons/icon-funnel-filled.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-funnel-filled.tsx:1]: the real change

File: src/generated/mistica-icons/icon-funnel-light.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-funnel-light.tsx:1]: the real change

File: src/generated/mistica-icons/icon-funnel-regular.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-funnel-regular.tsx:1]: the real change

File: packages/import-mistica-icons/yarn.lock
Comment (yceballost) [inline packages/import-mistica-icons/yarn.lock:1]: is this change expected?

**Review by marcoskolodny — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

File: packages/import-mistica-icons/yarn.lock
Comment (pladaria) [reply packages/import-mistica-icons/yarn.lock:1]: It is be safe to commit but I don't know what produced this change.

This file is updated when you change dependencies

**Pull request merged**